packages/material-ui-system/src/grid.js

```diff
@@ -50,7 +50,7 @@ export const gridArea = style({
 });
 
 const grid = compose(
-  gridGap,
+  gap,
   gridColumnGap,
   gridRowGap,
```

Comment:
In theory, it seems that the new names for these are:

```suggestion
  columnGap,
  rowGap,
```